pkg/templates/cluster_deployment_template.go

```diff
@@ -37,14 +39,110 @@ func (t clusterDeploymentTemplate) GetName() string {
 	return t.template.Name
 }
 
-func (t clusterDeploymentTemplate) GetOutput(stampedObject *unstructured.Unstructured) (*Output, error) {
-	return &Output{}, nil
+func (t clusterDeploymentTemplate) GetOutput(stampedObject *unstructured.Unstructured, templatingContext map[string]interface{}) (*Output, error) {
+	if err := t.outputReady(stampedObject); err != nil {
+		return nil, err
```

Comment:
not sure if intended or not - should this err be wrapped?

Reply:
Better to leave it unwrapped. Carto throws typed errors if the output is not ready (e.g. the conditions on the Deployment are not met) and wrapping means that our later error handling must unwrap them further.